editor/gui/output_dock.h

```diff
@@ -32,3 +32,11 @@ class OutputDock
 	OutputDockSettings& getSettings() { return m_OutputDockSettings; }
 	void setActive(bool enabled) { m_OutputDockSettings.m_IsActive = enabled; }
 };
+
+enum class OutputState : unsigned int
```

Comment:
Should be placed inside the OutputDock class

Reply:
Shifted enum class OutputState inside OutputDock class